recipe/patch_yaml/ray-packages.yaml

```diff
@@ -0,0 +1,19 @@
+---
+# https://github.com/conda-forge/ray-packages-feedstock/issues/147
+if:
+  name: ray-data
+  version_le: 2.9.1
+  timestamp_lt: 1706823371000
+then:
+  - replace_depends:
+      old: pandas?( *)
+      new: pandas >=1.3,<2.2*
+---
+if:
+  name: ray
```

Comment:
is this meant to be here? afaik no `ray` package exists on conda-forge.

Reply:
nope, older versions didn't have a pandas version specifier so I was trying to handle that but I just hardcoded in `new`